tests/compile-test.rs

```diff
@@ -166,7 +69,8 @@ fn base_config(test_dir: &str) -> compiletest::Config {
 
 fn run_ui() {
     let mut config = base_config("ui");
-    config.rustfix_coverage = true;
+    config.program.args.push("-Dwarnings".into());
```

Comment:
Could this be moved into `base_config`? So that there's a smaller diff for `ui-toml` tests that emit lints